Docs.md

```diff
@@ -145,7 +145,7 @@ browser JavaScript, Deno can import libraries directly from URLs. This example
 uses a URL to import a test runner library:
 
 ```ts
-import { test, assertEqual } from "https://deno.land/x/testing/testing.ts";
+import { test, assertEqual } from "https://deno.land/x/testing/mod.ts";
```

Comment:
⚠️ There's a `std/` missing here and below, it needs to be
```suggestion
import { test, assertEqual } from "https://deno.land/x/std/testing/mod.ts";
```

Reply:
hmmm -- Do I need to wait, then, for registry being deployed or anything? I'm wondering since it doesn't seem to work for me right now:
```
Compiling /Users/stephan/Misc/deno/test.ts
Downloading https://deno.land/x/testing/mod.ts... NOT FOUND
NotFound: Cannot resolve module "https://deno.land/x/testing/mod.ts" from "/Users/stephan/Misc/deno/test.ts"
    at DenoError (deno/js/errors.ts:19:5)
    at maybeError (deno/js/errors.ts:38:12)
    at maybeThrowError (deno/js/errors.ts:26:15)
    at sendSync (deno/js/dispatch.ts:67:5)
    at codeFetch (deno/js/os.ts:39:19)
    at _resolveModule (deno/js/compiler.ts:235:38)
    at moduleNames.map.name (deno/js/compiler.ts:515:31)
    at resolveModuleNames (deno/js/compiler.ts:507:24)
    at compilerHost.resolveModuleNames (deno/third_party/node_modules/typescript/lib/typescript.js:32750:71)
    at resolveModuleNamesWorker (deno/third_party/node_modules/typescript/lib/typescript.js:20688:69)
```

Reply:
```
deno test.ts --reload
```

Reply:
Sorry, this doesn't work.

Reply:
heh, yup. I wonder if someone needs to run [`publish.sh`](https://github.com/denoland/registry/blob/master/deploy.sh)...

Reply:
yes, sorry. I need to automate it...

Reply:
You could use [now](https://now.sh) or [Netlify](https://netlify.com). If you use Netlify, you could avoid having a server by using @kentcdodds’s [`netlify-shortener`](https://GitHub.com/kentcdodds/netlify-shortener)